website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -339,13 +339,13 @@ function Clock(props) {
 
 ### Imports {#imports}
 
-:::caution react-live and imports
+:::info globals and imports
```

Comment:
Not sure if we still need this section ;p

Reply:
why? this remains true no?